cmd/capdctl/main.go

```diff
@@ -276,7 +264,20 @@ func apply(cfg *rest.Config, objs []runtime.Object) error {
 		return err
 	}
 
-	for _, obj := range objs {
+	namespaces, remainingObjects := extractNamespaces(objs)
+	for _, ns := range namespaces {
```

Comment:
maybe break this out into a function, since it's been duplicated?

Reply:
i figured out how to do it in kustomize. this stuff is all deleted